src/main/java/org/sonar/plugins/ldap/LdapAuthenticator.java

```diff
@@ -95,6 +96,7 @@ private boolean checkPasswordUsingBind(String principal, String password, String
     InitialDirContext context = null;
     try {
       context = contextFactories.get(ldapKey).createUserContext(principal, password);
+	  LOG.debug("Password valid for user {} in server {}!", new Object[] {principal, ldapKey});
```

Comment:
Master branch uses now SonarQube Loggers, so this statement should be replaced:
`LOG.debug("Password valid for user {} in server {}!", principal, ldapKey);`